Content.Shared/Chat/TypingIndicator/SharedTypingIndicatorSystem.cs

```diff
@@ -12,6 +12,7 @@ public abstract class SharedTypingIndicatorSystem : EntitySystem
     /// </summary>
     [ValidatePrototypeId<TypingIndicatorPrototype>]
     public const string InitialIndicatorId = "default";
+    private string _originalIndicator = "default";
```

Comment:
1. do not store this on the system, store it on the component
2. remove the now unused InitialIndicatorId

Reply:
OK this is gonna require a lot of work (Fair enough) so it might be a while 😭